tensorflow/cc/gradients/math_grad.cc

```diff
@@ -687,6 +687,22 @@ Status MeanGrad(const Scope& scope, const Operation& op,
 }
 REGISTER_GRADIENT_OP("Mean", MeanGrad);
 
+Status ErfGrad(const Scope& scope, const Operation& op,
+               const std::vector<Output>& grad_inputs,
+               std::vector<Output>* grad_outputs) {
+  auto grad = grad_inputs[0];
+  auto two_over_root_pi = Cast(scope, Const(scope, 2 / std::sqrt(M_PI)),
```

Comment:
It seems that some platforms don't have M_PI,  I found `pi` is wrote as below:

```c++
  const double pi = std::atan(1) * 4;
```

Reply:
If you can use constexpr that's better.

Reply:
I just wonder whether an unified solution for `pi` exists.

Reply:
Looks like quite a lot of code uses M_PI. I would just use that for now; if we run into the issue that M_PI is missing in the future we can come up with a more permanent solution. Thanks.